test/table.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {mount, configure} from 'enzyme';
+import Table from '../src/client/Table';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Instead of setting `configure` in front of every test suite, I suggest to set the config before our test and then we don't have to do the same thing over and over.